03/app.js

```diff
@@ -2,35 +2,15 @@ export default function randomNumber(min, max) {
     min = Math.ceil(min);
     max = Math.floor(max);
 
-    console.log(min)
-    console.log(max)
-
     // both parameters must be numbers
     if (typeof min !== 'number' || isNaN(min) || typeof max !== 'number' || isNaN(max)) {
-        console.log('Both parameters must be numbers!')
-    }
-
-    // parameter min must be less or equal as parameter max
-    if (min > max) {
-        min = max;
+       console.log('Both parameters must be numbers!');
```

Comment:
Spokojnie można by było zwrócić np. `null` lub `NaN` - aby móc to przetestować.
Sam komunikat obsłużylibyśmy poza tą funkcją.